superset-frontend/packages/superset-ui-core/src/utils/featureFlags.ts

```diff
@@ -52,7 +52,6 @@ export enum FeatureFlag {
   SqlValidatorsByEngine = 'SQL_VALIDATORS_BY_ENGINE',
   SshTunneling = 'SSH_TUNNELING',
   TaggingSystem = 'TAGGING_SYSTEM',
-  ThemeEnableDarkThemeSwitch = 'THEME_ENABLE_DARK_THEME_SWITCH',
```

Comment:
Just confirmed. This is meant to be here. Not a leftover. Please, check https://github.com/apache/superset/pull/33420/files

Reply:
Gotcha. Felt redundant as switching to dark can be achieved in a few extra clicks. I don't mind keeping it.